### PR TITLE
Combine jobs by OS on the integration test

### PR DESCRIPTION
**Pull request by jhs7jhs**
Now it can be put together. https://github.com/CodeChain-io/codechain/commit/c5f8555dd034989e78d122bddb22b0be66ad7ed4

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,38 +12,34 @@ jobs:
...
+        - nvm install 8
+        - nvm use 8
+        - npm install -g yarn
         - rustup toolchain install nightly-2018-07-17
         - rustup component add rustfmt-preview --toolchain nightly-2018-07-17
       before_script:
         - cargo fetch --verbose
       script:
-        - cargo +nightly-2018-07-17 fmt -- --check && RUST_BACKTRACE=1 cargo test --verbose --all
+        - cargo +nightly-2018-07-17 fmt -- --check
```
Comment (sgkim126) [inline .travis.yml]: AFAIK, It uses `&&` since Travis runs the next script even the previous script failed.
Would you check it?

Comment (sgkim126) [reply .travis.yml]: https://github.com/sgkim126/travis-test/commit/4edea17a19845da5cecc451e28019d0c1f7227bc
https://travis-ci.org/sgkim126/travis-test/jobs/440060659 Yes it does.

Comment (jhs7jhs) [reply .travis.yml]: I think it should be in a one line, for fast termination. I'll fix this.

Comment (jhs7jhs) [reply .travis.yml]: I'll fix this using a flag env variable

**Pull request merged**